test/extensions/quic_listeners/quiche/platform/quic_platform_test.cc

```diff
@@ -96,9 +97,9 @@ class QuicPlatformTest : public testing::Test {
 
 TEST_F(QuicPlatformTest, QuicAlignOf) { EXPECT_LT(0, QUIC_ALIGN_OF(int)); }
 
-TEST_F(QuicPlatformTest, QuicArraysize) {
+TEST_F(QuicPlatformTest, QuicheArraysize) {
```

Comment:
Can we move QuicheArraysize and QuicheStringPiece to a new quiche_platform_test.cc?

Reply:
done